app/script_variant.py

```diff
@@ -44,9 +44,9 @@ def has_kanji(word):
     return re.match(ur'([\u3400-\u9fff])', word)
 
 
-def romanize_japanese_name_by_name_dict(word):
+def romanize_single_japanese_word_by_name_dict(word):
     """
-    This method romanizes japanese name by using name dictionary.
+    This method romanzies japanese name by using name dictionary.
```

Comment:
s/romanizies/romanizes/

Also you should mention that this method doesn't support romanizing full names using first/last names in the dictionary. Otherwise it's not clear how this method is different from romanize_japanese_name_by_name_dict().